.github/workflows/quicktest.yml

```diff
@@ -0,0 +1,79 @@
+name: Quick Tests
+
+on:
+  push:
+    branches-ignore:
+      - master
+      - develop
+
+jobs:
+  # Performs some quick tests.
+  # This is a much quicker test suite which only runs the unit tests and linting
+  # against the low/high supported PHP/PHPCS combinations.
+  #
+  # Performs the following steps:
+  # - Checks out the repository.
+  # - Configures caching for Composer.
+  # - Sets up PHP.
+  # - Installs PHPCS.
+  # - Install Composer dependencies.
+  # - Runs the Composer linting script (if desired).
+  # - Runs Composer run-tests script.
+  quick-tests:
+    name: PHP ${{ matrix.php }} on PHPCS ${{ matrix.phpcs_branch }}${{ matrix.lint && ' with linting' || '' }}
```

Comment:
Scratch that second point - I just noticed that the workflow name is prefixed to the job name by default if there are multiple workflows. So this would only apply to single workflow repos and multi-workflow with multiple jobs in a single workflow.

Reply:
Actually, in the "required status" overview, the workflow name is not prefixed, so it would be impossible to tell the difference between the `quicktest` and the `test` workflow when setting the statuses, so my original remark stands.

Reply:
Ah, thanks for catching this! I originally had a `lint` key in the matrix, but then I removed it because linting should always run for the quick test and unit test workflows, so it was unnecessary.